neurons/script/tests/test_script.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+import os
+
+from core.NeuronModule import MissingParameterException, InvalidParameterException
+from neurons.script.script import Script
+from core.FileManager import FileManager
+
+
+class TestScript(unittest.TestCase):
+
+    def setUp(self):
+        self.path="path"
+        self.random = "random"
+
+    def testParameters(self):
+        def run_test_missing_param(parameters_to_test):
+            with self.assertRaises(MissingParameterException):
+                Script(**parameters_to_test)
+
+        def run_test_invalid_param(parameters_to_test):
+            with self.assertRaises(InvalidParameterException):
+                Script(**parameters_to_test)
+
+        # empty
+        parameters = dict()
+        run_test_missing_param(parameters)
+
+        # missing path
+        parameters = {
+            "random": self.random
+        }
+        run_test_missing_param(parameters)
+
+        # ramdom path
+        self.path="/tmp/iamarandompath/anotherrandompath/kalliope"
+        parameters = {
+            "path": self.path
+        }
+        run_test_invalid_param(parameters)
+
+        # Test Non executable file
+        # Create the file and remove permissions to the user
+        tmp_file_path = "/tmp/kalliope/tests/neuronScript"
+        FileManager.write_in_file(tmp_file_path, "[kalliope-test] TestScript - testParameters")
+        os.chmod(tmp_file_path, 0600)
```

Comment:
ERROR:kalliope:I/O error(2): No such file or directory
E
======================================================================
ERROR: testParameters (__main__.TestScript)
----------------------------------------------------------------------
Traceback (most recent call last):
 File "/home/nico/Documents/kalliope/neurons/script/tests/test_script.py", line 45, in testParameters
   os.chmod(tmp_file_path, 0600)
OSError: [Errno 2] No such file or directory: '/tmp/kalliope/tests/neuronScript'
depuis pycharm
je me suis dit direct que ça allait faire ça
normal  le dossier exist pas